docs/_data/nakadi-event-bus-api.yaml

```diff
@@ -822,6 +822,46 @@ paths:
           schema:
             $ref: '#/definitions/Problem'
 
+  '/event-types/{name}/partitions-number':
```

Comment:
The name `partitions-number` is a confusing approximation for `number-of-partitions`.
I suggest one of `partition-count`, `partition-configs`, `partition-status` ....

Reply:
91ea632b2 👍